lang/test/org/partiql/lang/errors/ParserErrorsTest.kt

```diff
@@ -1752,6 +1752,8 @@ class ParserErrorsTest : SqlParserTestBase() {
         )
     }
 
+    // Some of the ORDER BY related tests are changed
+    // because the window function adds an additional rule that uses ORDER BY.
```

Comment:
I had a conversation with John to discuss this. Per my understanding we are considered backward compatible since we are still throwing ParserError upon seeing wrong syntax.